cf-Invalidation-service/index.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+/*
+*
+*   Title : Invalidation Service for Covid Alert QA Tools
+*   Purpose : Receive an API call to trigger invalidation.
+*   Author : Timothy Patrick Jodoin
+*   Title : Principal Solution Architect
+*   Email : tj@fsdcsolutions.com
+*   Firm : FSDC Solutions Inc.
+*   Date : Jan 12th, 2021
+*   Client : CDS
+*
+*/
+
+const
+    AWS = require( 'aws-sdk' ),
+    uuid = require( 'uuid' ),
+    CF = new AWS.CloudFront();
+    AWS.config.update({region: 'ca-central-1'});
+
+exports.handler = async event => {
+
+    var distribution_ID = process.env.distributionID;   //encrypted env_var
+    let callerReference = uuid.v1();  // RFC4122 standard uuid Timestamp based generation
+
+// populate invalidation object
+    console.log("ID :",distribution_ID);
```

Comment:
We should be able to inspect this through the lambda configuration and we don't need to log it here.

Reply:
Sorry. This was for debug and security audits from a consolidated bucket. Yes this will be visible in lambda and parameter store. But in the event it is looked at from cloud watch these values won't be visible.

Reply:
Sounds good.